client/src/components/gamepage/Gamepage.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { } from 'react';
+import PropTypes from 'prop-types';
+import Sidebar from '../sidebar/Sidebar';
+import Roulette from '../roulette/Roulette';
+import ScratchTicket from '../scratch-ticket/ScratchTicket';
+import Slots from '../slots/Slots';
+
+function Gamepage({ user, game }) {
+  const renderGame = () => {
+    if (game === 'roulette') {
+      return <Roulette />;
+    }
+    if (game === 'scratch-ticket') {
+      return <ScratchTicket />;
+    }
+    return <Slots />;
+  };
+
+  return (
+    <div className="game-container">
+      <Sidebar user={user} />
+      {renderGame()}
+    </div>
+  );
+}
+
+Gamepage.propTypes = {
+  user: PropTypes.object.isRequired,
```

Comment:
this line is throwing a linter error.
Our linter is configured to disallow generic objects as a prop type.
if you want to ignore this rule (which is what I would do, its an annoying rule) add
```
"react/forbid-prop-types": "off"
```
to the "rules" object in `.eslintrc.json`